roles/lib_openshift/library/oc_adm_ca_server_cert.py

```diff
@@ -413,7 +419,7 @@ def load(self, content_type='yaml'):
 
                 # Try to use RoundTripLoader if supported.
                 try:
-                    self.yaml_dict = yaml.safe_load(contents, yaml.RoundTripLoader)
+                    self.yaml_dict = yaml.load(contents, yaml.RoundTripLoader)
```

Comment:
Just verifying, this is so you can use  either ``yaml`` library, correct?

Reply:
This should probably be safe_load as well but yes, either library can be used.

Reply:
Actually, tested this with ruamel and safe_load does not have a `Loader` kw param.

Reply:
> Actually, tested this with ruamel and safe_load does not have a Loader kw param.

boo! Ok fair enough 😄